lisa/transformers/kernel_source_installer.py

```diff
@@ -39,6 +38,12 @@ class LocalLocationSchema(BaseLocationSchema):
             required=True,
         ),
     )
+    config_path: str = field(
```

Comment:
Move it to BaseLocationSchema, so it doesn't need similar implementation on subclasses.

Reply:
sure, Chi

Reply:
done